modules/core/src/main/java/org/apache/ignite/internal/IgniteFeatures.java

```diff
@@ -82,7 +82,13 @@
     /**
      * A mode when data nodes throttle update rate regarding to DR sender load
      */
-    DR_DATA_NODE_SMART_THROTTLING(19);
+    DR_DATA_NODE_SMART_THROTTLING(19),
+
+    /**
```

Comment:
You should implement unit test for new WC_DR_EVENTS feature, in ggprivate repository in pair branch